doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -5,21 +5,91 @@
 
 ## Table of Contents
 * [1.1 Introduction.](#1.1)
-* [1.2 Terminology.](#1.2)
-* [1.3 Approach](#1.3)
-* [1.4 Principles](#1.4)
-* [1.5 scope](#1.4)
-* [1.6 Vision](#1.4)
-* [1.7 Roadmap](#1.4)
+* [1.2 Terminology](#1.2)
+* [1.3 Principles](#1.3)
+  * [1.3.1 Cloud Native.](#1.3.1)
+* [1.4 Scope](#1.4)
+* [1.5 Vision](#1.5)
+* [1.6 Approach](#1.6)
+* [1.7 Roadmap](#1.7)
+
 
 <a name="1.1"></a>
 ## 1.1 Introduction
 
-This document (at this stage, not a RA, we're just using the template as a placeholder), discusses the role of Kubernetes in managing NFV software components (e.g. VNFCs). Application containers such as those managed by Kubernetes can be considered a Foundational PaaS capability<sup>1</sup> and so the following terms can all be thought of as being the topic of this document:
-- Kubernetes PaaS (Platform-as-a-Service)
-- CaaS (Containers-as-a-Service)
-- Kubernetes-based CaaS (Containers-as-a-Service)
-- Kubernetes-as-a-Service
+> to be written
+
+
+<a name="1.2"></a>
+### 1.2 Terminology
+
+|Term|Definition|Reference|
+|---|---|---|
+|Container Image|A container image is a lightweight, standalone, executable package of software that includes everything needed to run an application: code, runtime, system tools, system libraries and settings.|[Link](https://www.docker.com/resources/what-container)|
+|Container|A running instantiation of a container image.|-|
+|Pod|A Pod (as in a pod of whales or pea pod) is a group of one or more containers (such as Docker containers), with shared storage/network, and a specification for how to run the containers. A Pod’s contents are always co-located and co-scheduled, and run in a shared context.|[Link](https://kubernetes.io/docs/concepts/workloads/pods/pod/)|
+|Cluster|A cluster consists of at least one cluster master and multiple worker machines called nodes. These master and node machines run the Kubernetes cluster orchestration system.|[Link](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture)|
+|Master|The cluster master runs the Kubernetes control plane processes, including the Kubernetes API server, scheduler, and core resource controllers.|[Link](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture)|
+|Node|Nodes are the worker machines (physical or virtual) that run your containerised applications and other workloads. Each node is managed from the master, which receives updates on each node's self-reported status. A node runs the services necessary to support the Docker containers that make up your cluster's workloads. These include the Docker runtime and the Kubernetes node agent (kubelet) which communicates with the master and is responsible for starting and running Docker containers scheduled on that node.|[Link](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture)|
```

Comment:
- Lets use "container runtime" instead of Docker. 
- Should be formulated in passive mode. 
- kubelet is responsible for the whole LCM of the container runtimes.

Reply:
https://kubernetes.io/ja/docs/reference/glossary/?fundamental=true#term-node

I agree we should keep this technology neutral ("container runtime" instead of "Docker runtime" and "OS containers" instead of "Docker containers"), as it applies to any CRI/OCI compliant implementation. In this context, it'd be more correct to talk about pods instead of containers anyway.

Reply:
May I suggest that we also tie the "node" to "pods" as in "A node hosts one or more pods.